src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/BlazorDebug/Events.ts

```diff
@@ -0,0 +1,35 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import * as psList from 'ps-list';
+import { DebugSession } from 'vscode';
+
+import { RazorLogger } from '../RazorLogger';
+
+export async function onDidTerminateDebugSession(
+  event: DebugSession,
+  logger: RazorLogger,
+  targetPid: number | undefined,
+) {
+  logger.logVerbose('Terminating debugging session...');
+  try {
+    const processes = await psList();
+    if (targetPid) {
```

Comment:
At the beginning of this method should we just do:

```TypeScript
if (!targetPid) {
    return;
}
```

That way we can avoid listing processes if we wont be able to do anything with em anyhow. 

Also, is process ID 0 valid? If so the truthy if check will also return false I believe

Reply:
Fixed.

We shouldn't expect a PID 0 so this check should cover the relevant cases.

Reply:
> We shouldn't expect a PID 0 so this check should cover the relevant cases.

Is PID 0 reserved?